features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/tracking/AndroidXFragmentLifecycleCallbacks.kt

```diff
@@ -57,23 +58,17 @@ internal open class AndroidXFragmentLifecycleCallbacks(
     }
 
     // endregion
+    override fun onFragmentViewCreated(fm: FragmentManager, f: Fragment, v: View, savedInstanceState: Bundle?) {
```

Comment:
`onFragmentActivityCreated` is now deprecated and according to comment [here](https://developer.android.com/jetpack/androidx/releases/fragment#1.3.0-alpha02) it seems we can safely move its logic into `onFragmentViewCreated`.

@xgouchet, please let me know if I'm mistaken.

Reply:
That looks good to me 👍